### ISSUE TITLE
[Suggestion] WorldBorder Integration

### ISSUE THREAD

**Issue by BlackBeltPanda** [rlf/uSkyBlock#914]
I'd love to see a form of WorldBorder integration, ideally integration with both the vanilla world border and the [WorldBorder plugin](https://dev.bukkit.org/bukkit-plugins/worldborder/) with a config setting to toggle one or the other.

The idea is that uSkyBlock would create a world border at the edge of the furthest island's region, centered at 0,0, and expand as necessary. This would also be beneficial in preventing chunks from being loaded by players with the ability to fly who decide to fly straight for hours on end. It would also help in the off chance that terrain generation fails and vanilla terrain is generated (has happened to me a few times), preventing flying players from generating tons of vanilla terrain that has to be WorldEdit'ed away.

**Comment by Lmmb74** [comment]
So, maybe doing some per player borders would be nice, right? I mean like using per player packets (ProtocolLib) to apply borders to player island, but only he/she can see them.

**Comment by rlf** [comment]
Ok, just to clarify your suggestion @BlackBeltPanda , you'd like the following:

  1. A config-setting allowing configuration of World-Border integration
  2. When config is enabled, maintain the world-border when a new island is created (increasing or decreasing the border as  neccesary).

Is this correct?

Reg. @Lmmb74 - we already have that, try out the `/usb rg show` commands.

**Comment by BlackBeltPanda** [comment]
That's correct; basically a way to prevent players from flying outside the area in which all islands exist.

**Comment by Lmmb74** [comment]
I can make this, but tell me @rlf which would be better:
NMS (without Reflection but declaring which to use in main class :P);
or Reflection?

**Comment by rlf** [comment]
As I said @Lmmb74 , we already have this! No need for you to "make it".
Checkout https://github.com/rlf/uSkyBlock/blob/master/uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/command/admin/RegionCommand.java

But, please open a new issue, if you'd like to discuss improvements, since this issue is about something completely different.
